pymeasure/experiment/results.py

```diff
@@ -40,12 +41,62 @@
 log.addHandler(logging.NullHandler())
 
 
+def replace_placeholders(string, procedure, date_format="%Y-%m-%d", time_format="%H:%M:%S"):
+    """ Replaces the placeholders in the provided strings with the values of the
+    associated parameters, as provided by the procedure. This uses the standard
```

Comment:
```suggestion
    """Replace placeholders in string with values from procedure parameters.
    
    Replaces the placeholders in the provided string with the values of the
    associated parameters, as provided by the procedure. This uses the standard
```
For docstrings it's customary to have the first line as a "short summary" in imperative tone.

Reply:
Thanks for this remark, I wasn't aware of this.